.github/workflows/release-windows.yml

```diff
@@ -201,10 +194,15 @@ jobs:
       # If this is run on a fork, there may not be a certificate set up - continue in this case
       continue-on-error: true
       run: |
-        cd ${GITHUB_WORKSPACE}/build/bundles
-        ../../os/windows/sign.bat *.exe "${WINDOWS_CERTIFICATE_COMMON_NAME}"
+        cd ${GITHUB_WORKSPACE}/os/windows
+        ./sign_azure.bat "${GITHUB_WORKSPACE}/build/bundles"
```

Comment:
Why not just call `pwsh -File ..` directly here? This level of indirection doesn't seem to add much value?

Reply:
I was basically just copying what we already had for the other PowerShell scripts in the repository!